content/docs/authorization/deployment/powermax/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: PowerMax
+linktitle: PowerMax
+description: >
+  Enabling CSM Authorization for PowerMax CSI Driver
+---
+
+## Configuring PowerMax CSI Driver with CSM for Authorization
+
+Given a setup where Kubernetes, a storage system, and the CSM for Authorization Proxy Server are deployed, follow the steps below to configure the CSI Drivers to work with the Authorization sidecar:
```

Comment:
Change "the steps below" to "these steps"

Reply:
fixed.